pkg/controller/operators/olm/requirements.go

```diff
@@ -3,11 +3,13 @@ package olm
 import (
 	"encoding/json"
 	"fmt"
+
 	"github.com/sirupsen/logrus"
 
 	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
 	olmErrors "github.com/operator-framework/operator-lifecycle-manager/pkg/controller/errors"
 	"github.com/operator-framework/operator-lifecycle-manager/pkg/controller/install"
+	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
```

Comment:
nit: import ordering

Reply:
I thought my tooling was out of date, but I just did a `go get golang.org/x/tools/cmd/goimports` and ran goimports on requirements.go and it still put the import line in the same spot. Where does yours get put?